### ISSUE TITLE
Unparsing into string array

### ISSUE THREAD

**Issue by buddhikac96** [commandlineparser/commandline#665]
In the project I am working there is a method which need command line arguments as a `string array` which I can't change.  So how to unparse into string array. 
I can get a string by unparsing using `var arguments = Parser.Default.FormatCommandLine(Program.cliOption)`. 

But my argument will be something like this.
`app install --path "c:/My folder/app" --validate = false`

So if I use above workaround I can't use `string.Split(' ')` since it possible to contains `spaces`  in file paths. So is there any workaround to get string array from the `object`?

**Comment by moh-hassan** [comment]
The develop branch contain a new method `FormatCommandLineArgs` that generate array of strings.
You can download the package [LineParser.2.8.0-ci-176.nupkg](https://ci.appveyor.com/api/buildjobs/ey2q07iwq2ic9x19/artifacts/src%2FCommandLine%2Fbin%2FRelease%2FCommandLineParser.2.8.0-ci-176.nupkg) from CI server.
This feature will be available in v2.9
 ```cs
 var options = new Options { FilePath = "c:/My folder/app", Validate = false };
 string[] arguments = Parser.Default.FormatCommandLineArgs(options);
```	
Output array:	
```
--path
c:/My folder/app
```

**Comment by buddhikac96** [comment]
@moh-hassan answer is clear. Thank you so much and closing the Issue.
